AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -214,9 +234,11 @@ - (CGRect)frame
 
 - (void)setFrame:(CGRect)rect
 {
-  _bridge_prologue;
-
-  if (_flags.synchronous && !_flags.layerBacked) {
+  _bridge_prologue_write;
+  BOOL setFrameDirectly = _flags.synchronous && !_flags.layerBacked;
+  BOOL isMainThread = ASDisplayNodeThreadIsMain();
+  BOOL nodeLoaded = self.nodeLoaded;
```

Comment:
This method is tremendously performance sensitive. Can you access the instance variable for nodeLoaded directly?